### ISSUE TITLE
Any way to add arrow key accessibility?

### ISSUE THREAD

**Issue by nickfortyau** [hoaphantn7604/react-native-element-dropdown#228]
I know this is mainly for native, but we're using this component on react-native plus web. On web we need the accessibility support so that arrow keys can navigate the dropdown menu. Does there exist a way to support that which I'm missing?
